qiskit/circuit/gate.py

```diff
@@ -26,7 +26,12 @@ class Gate(Instruction):
     """Unitary gate."""
 
     def __init__(
-        self, name: str, num_qubits: int, params: List, label: Optional[str] = None
+        self,
+        name: str,
+        num_qubits: int,
+        params: List,
+        label: Optional[str] = None,
+        condition: Optional[Tuple] = None,
```

Comment:
The new docstring for "condition" needs to be here as well.

Reply:
added